crates/bevy_wgpu/src/lib.rs

```diff
@@ -119,10 +108,12 @@ pub fn get_wgpu_render_system(resources: &mut Resources) -> impl FnMut(&mut Worl
 }
 
 #[derive(Default, Clone)]
-pub struct WgpuOptions {
+pub struct WgpuOptions<'a> {
+    pub name: Option<&'a str>,
```

Comment:
Can we make `name` a `Cow<'static, str>`? I don't want  "plugin configuration" types to have lifetimes.

Reply:
Done!